common/attachments/index.ts

```diff
@@ -50,4 +61,60 @@ export async function getAttachmentURL(attachmentId: string, key: string, attach
         attachmentGroupId = dbAttachment.attachmentGroupId;
     }
     return await generatePresignedURL(`${attachmentGroupId}/${attachmentId}/${key}`, ATTACHMENT_BUCKET);
+}
+
+
+/**
+ * Function creating the HTML which is attached to the email, consisting of a headline and a bulleted list with the attachment links, file names and file sizes.
+ * @param attachments   The attachments to be added to the list.
+ * @param attachmentGroupId
+ * @return              HTML of the list.
+ */
+export async function getAttachmentListHTML(attachments: { attachmentId: string, filename: string, size: number }[], attachmentGroupId: string) {
+    let attachmentListHTML = "<h3>Anhänge</h3>";
+
+    for (let {attachmentId, filename, size} of attachments) {
+        let key;
+        if(!filename || !size) {  // When sending reminders, for example, we don't have immediate access to data like size and the filename, we only have the attachmentGroupId
+            logger.warn("Fetching data for attachment with ID " + attachmentId);
```

Comment:
Who is supposed to be warned? Also how can the filename be undefined if the functions signature suggests that it is present?